tools/nnyeah/nnyeah/MethodTransformations.cs

```diff
@@ -0,0 +1,832 @@
+					Instruction.Create (OpCodes.Sub)
+				}
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_Addition(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Add)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_Subtraction(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Sub)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_Multiply(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Mul)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_Division(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Div_Un)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_Modulus(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Rem_Un)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_BitwiseAnd(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.And)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_BitwiseOr(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Or)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_ExclusiveOr(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Xor)
+				),
+			new Transformation(
+				"System.nuint System.nuint::op_LeftShift(System.nuint,System.Int32)",
+				Instruction.Create (OpCodes.Shl)
+				),
+			new Transformation (
+				"System.nuint System.nuint::op_RightShift(System.nuint,System.Int32)",
+				Instruction.Create (OpCodes.Shr)
+				),
+			new Transformation(
+				"System.Boolean System.nuint::op_Equality(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Ceq)
+				),
+			new Transformation(
+				"System.Boolean System.nuint::op_Inequality(System.nuint,System.nuint)",
+				TransformationAction.Replace,
+				new List<Instruction> () {
+					Instruction.Create (OpCodes.Ceq),
+					Instruction.Create (OpCodes.Ldc_I4_0),
+					Instruction.Create (OpCodes.Ceq)
+				}
+				),
+			new Transformation(
+				"System.Boolean System.nuint::op_LessThan(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Clt_Un)
+				),
+			new Transformation(
+				"System.Boolean System.nuint::op_GreaterThan(System.nuint,System.nuint)",
+				Instruction.Create (OpCodes.Cgt_Un)
+				),
+			new Transformation(
+				"System.Boolean System.nuint::op_LessThanOrEqual(System.nuint,System.nuint)",
+				TransformationAction.Replace,
+				new List<Instruction> () {
+					Instruction.Create (OpCodes.Cgt_Un),
+					Instruction.Create (OpCodes.Ldc_I4_0),
+					Instruction.Create (OpCodes.Ceq)
+				}
+				),
+			new Transformation(
+				"System.Boolean System.nuint::op_GreaterThanOrEqual(System.nuint,System.nuint)",
+				TransformationAction.Replace,
+				new List<Instruction> () {
+					Instruction.Create (OpCodes.Clt_Un),
+					Instruction.Create (OpCodes.Ldc_I4_0),
+					Instruction.Create (OpCodes.Ceq)
+				}
+				),
+			new Transformation (
+				"System.Void System.nuint::.cctor()"
+				),
```

Comment:
Could you have done something very similar but without the new Transform in every line? Add a dict with a key value pair, loop and call the constructor.

Reply:
Now that I look at it again, I believe that you do not need the Transformation there, and a simple new () would have worked.